website/content/docs/enterprise/network-segments.mdx

```diff
@@ -11,19 +11,199 @@ description: |-
 <EnterpriseAlert>
   This feature requires{' '}
   <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>{' '}
-  with the Global Visibility, Routing, and Scale module.
+  version 0.9.3+ with the Global Visibility, Routing, and Scale module.
 </EnterpriseAlert>
 
-Consul Network Segments enables operators to create separate LAN gossip segments
-in one Consul cluster. Agents in a segment are only able to join and communicate
-with other agents in its network segment. This functionality is useful for
-clusters that have multiple tenants that should not be able to communicate
-with each other.
+Consul requires full connectivity between all agents (servers and clients) in a
+datacenter within a given LAN gossip pool. By default, all Consul agents will be
+a part of one shared LAN gossip pool known as the `<default>` network segment,
+thus requiring full mesh connectivity within the datacenter.
 
-To get started with network segments you can review the tutorial on HashiCorp Learn for
-[Network Segments](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters).
+![Consul datacenter default agent connectivity: one network segment](/img/network-segments/consul-network-segments-single.png)
 
-~> **Note:** Prior to Consul 1.7.3, a Consul agent configured with too many network segments may not be able to start due to [limitations](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters#network-segments-limitations) in Serf.
+But many advanced Consul users have the need to run a datacenter with a segmented
+network, meaning that there is not full connectivity between all agents. This is
+usually the result of business policies enforced via network rules or firewalls.
+To use Consul with a segmented network, break up the LAN gossip pool along
+network segment boundaries into "network segments": isolated LAN gossip pools
+with full connectivity between all agent members.
+
+![Consul datacenter agent connectivity with network segments](/img/network-segments/consul-network-segments-multiple.png)
+
+To get started with network segments you can review the tutorial on HashiCorp
+Learn for [Network Segments](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters).
+
+-> **Info:** Network segments enable operating a datacenter without full mesh
```

Comment:
I agree with changing `operating` to `you to operate`, but I find the meaning less clear if I switch `without full mesh connectivity` to `with limited mesh connectivity`.

Reply:
It's your call. For many people, concepts can be easier to consume when they are described in terms of presence -- _having_ something instead of _not having_ something. That's why I suggested the 'without => with' change. Perhaps 'limited' isn't concrete enough. I defer to you.

Reply:
I generally agree. In this case, my understanding is that "mesh" = "everything is connected", so "limited mesh" isn't a clear phrase. I'll spend a few minutes seeing if I can switch the statement to a "having something" statement with different wording. If not, I'll keep as is.